src/App.js

```diff
@@ -1,3 +1,18 @@
-export default function App() {
-  return <h1>Beads And Beyonds</h1>;
+import "./App.css";
+import { Navbar, Footer, Carousel, Aside, Category } from "./component";
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
Are you using this App className is css or not? If not then use fragment tah insted of this div